api/controllers/user.js

```diff
@@ -0,0 +1,29 @@
+const db = require("../models/index");
+
+const Users = db.users;
+const { hashPassword, isPasswordValid, generateToken } = require("../helpers/authHelper");
+
+exports.signup = async (req, res) => {
+  try {
+    // get values from body
+    const {
+      username, email, password, confirmPassword
+    } = req.body;
+    // checks if password and confirmPasword matche
+    if (password !== confirmPassword) {
+      return res.status(403).json({ response: "confirm password not matched with password" });
```

Comment:
Typo

Reply:
Where

Reply:
Okay

Reply:
You can use 'Invalid Password' No too much story